src/App.vue

```diff
@@ -255,7 +255,7 @@ export default {
 
     /** Get banner text. */
     bannerText (): string {
-      const bannerText: string = getFeatureFlag('banner-text')
+      const bannerText: string = GetFeatureFlag('banner-text')
```

Comment:
To me, these "feel right" in PascalCase since they're external imports instead of local methods (and they're not attached to `this`).